deploy/build.sh

```diff
@@ -3,8 +3,11 @@
 # Exit on any error
 set -e
 
-# Initialize build type variable as empty to ensure it's set by the user
+# Default values and color codes
 BUILD_TYPE=""
+BUILD_DIR="build"
```

Comment:
these color codes are different to those in existing deployment scripts, which makes the output a bit weird.